guidance/library/_block.py

```diff
@@ -7,7 +7,7 @@ def __init__(self, opener, closer, name=None):
         self.name = name
 
     def __enter__(self):
-        models.Model.open_blocks[self] = None
+        models.Model.open_blocks.add(self)
```

Comment:
This is clearly being used as a set but instead it was a dict of type Dict[self, None]

Reply:
It was not in fact a set XD. I found it used as a dictionary in other places.